exercises/nth-prime/example.jl

```diff
@@ -0,0 +1,43 @@
+function isPrime(x)
+	i = 2
+	if x == 1
+		return false
+	end
+	if x == 2
+		return true
+	end
+	if x == 3
+		return true
+	end
+	while i <= sqrt(x)
+		if x % i == 0
+		return false
+		end
+		i =i+1
+	end
+	return true
+end
+
+function nthPrime(n)
+	primeCount = 0
+	if n == 0 
+		return "there is no zeroth prime"

```

Comment:
This should not return a string, but raise an error instead, as per canonical-data:

```json
      "expected": {
        "error": "there is no zeroth prime"
      }
```

https://github.com/exercism/problem-specifications/blob/bd68e1be845b4c947fa9aa0dcd453d0428afcf18/exercises/nth-prime/canonical-data.json#L43-L45

The Julia manual has details on how to do that: https://docs.julialang.org/en/v1/manual/control-flow/#Exception-Handling-1

A `DomainError` fits this case best.